### ISSUE TITLE
Connections UI  - Can't compile project

### ISSUE THREAD

**Issue by Stanzarectah** [scottdurow/SparkleXrm#184]
When compiling I get the following:

Severity	Code	Description	Project	File	Line	Suppression State
Error	CS0246	The type or namespace name 'ES6' could not be found (are you missing a using directive or an assembly reference?)	ClientUI	C:\Users\Stanley\Downloads\SparkleXrm-master\SparkleXrm-master\SparkleXrmSamples\ConnectionsUI\ClientUI\ViewModel\QueryParser.cs	4	Active

Where can I get this reference from nuget?
When I type ES6 I get a bunch of results?
Which library is it?

**Comment by scottdurow** [comment]
Which branch are you using? It should be the master branch that you use.

**Comment by Stanzarectah** [comment]
Same thing happens for master branch.
I am using VS 2015

**Comment by Stanzarectah** [comment]
So anyone have the same issue?

I feel there is way to many issues way to many issues with this product go make it available for general public.

**Comment by scottdurow** [comment]
You are right - the updates that I've made recently to work with the V9 API have made their way into the master branch!  suggest you go back to https://github.com/scottdurow/SparkleXrm/tree/5b7f202239390718a624cc5b8761df7a69ec5b0d 
Sorry about that!
